src/app/components/named-entity/named-entity-detail/named-entity-detail.component.html

```diff
@@ -0,0 +1,36 @@
+<div [title]="'&lt;' + data.label + '&gt;' | lowercase">
+    <span [ngSwitch]="data.label" class="icon">
+        <evt-icon *ngSwitchCase="'actors'" [iconInfo]="{icon: 'users'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'birth'" [iconInfo]="{icon: 'birthday-cake'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'bloc'" [iconInfo]="{icon: 'map-marker'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'country'" [iconInfo]="{icon: 'map-marker'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'death'" [iconInfo]="{icon: 'times', rotate: 45}"></evt-icon>
+        <evt-icon *ngSwitchCase="'district'" [iconInfo]="{icon: 'map-marker'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'geogFeat'" [iconInfo]="{icon: 'map-marker'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'geoname'" [iconInfo]="{icon: 'map-marker'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'idno'" [iconInfo]="{icon: 'barcode'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'note'" [iconInfo]="{icon: 'sticky-note'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'occupation'" [iconInfo]="{icon: 'briefcase'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'orgname'" [iconInfo]="{icon: 'users'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'persname'" [iconInfo]="{icon: 'user'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'placename'" [iconInfo]="{icon: 'map-marker'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'region'" [iconInfo]="{icon: 'map-marker'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'relations'" [iconInfo]="{icon: 'share-alt'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'residence'" [iconInfo]="{icon: 'home'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'settlement'" [iconInfo]="{icon: 'location-arrow'}"></evt-icon>
+        <evt-icon *ngSwitchCase="'sex'" [iconInfo]="{icon: 'venus-mars'}"></evt-icon>
+        <evt-icon *ngSwitchDefault [iconInfo]="{icon: 'info-circle'}"></evt-icon>
```

Comment:
This can be simplified moving the data part inside the component and leaving here only something like:
```
<evt-icon [iconInfo]="iconData[data.label] || defaultIcon"></evt-icon>
```
and putting everything else inside:
```
iconData = {
  'actors': {icon: 'users'},
  ...
};

defaultIcon = {icon: 'info-circle'};
```